OpenShift/nuodb.3.2-openshift-persistent-template.yaml

```diff
@@ -0,0 +1,463 @@
+      spec:
+        labels:
+          group: nuodb
+          app: nuodb
+        storageClassName: ${PVC_CLASS_NAME}
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: ${SM_PVC_SIZE}
+- apiVersion: v1
+  kind: DeploymentConfig
+  metadata:
+    labels:
+      group: nuodb
+      app: nuodb
+      deploymentconfig: te-${NODE_REGION}
+    name: te-${NODE_REGION}
+  spec:
+    selector:
+      matchLabels:
+        affinity: te
+        app: te-${NODE_REGION}
+        deploymentconfig: te-${NODE_REGION}
+    replicas: ${TE_COUNT}
+    selector:
+      app: te-${NODE_REGION}
+      deploymentconfig: te-${NODE_REGION}
+      affinity: te
+    strategy:
+      activeDeadlineSeconds: 21600
+      resources: {}
+      rollingParams:
+        intervalSeconds: 1
+        maxSurge: 25%
+        maxUnavailable: 25%
+        timeoutSeconds: 600
+        updatePeriodSeconds: 1
+      type: Rolling
+    template:
+      metadata:
+        annotations:
+          openshift.io/generated-by: OpenShiftNewApp
+        labels:
+          app: te-${NODE_REGION}
+          deploymentconfig: te-${NODE_REGION}
+          affinity: te
+      spec:
+        nodeSelector:
+          zone: ${NODE_REGION}
+        affinity:
+          podAntiAffinity:
+            preferredDuringSchedulingIgnoredDuringExecution:
+            - weight: 100
+              podAffinityTerm:
+                labelSelector:
+                  matchExpressions:
+                  - key: affinity
+                    operator: In
+                    values:
+                    - te
+                topologyKey: "kubernetes.io/hostname"
+        containers:
+        - env:
+          - name: API_PORT
+            value: "8888"
+          - name: SUB_ID
+            value: VNB2L0GZW1
+          - name: PEER_ADDRESS
+            value: ${ADMIN_SVC_NAME}-${NODE_REGION}
+          - name: DB_NAME
+            valueFrom:
+              secretKeyRef:
+                name: ${SECRET_NAME}
+                key: database-name
+          - name: DB_USER
+            valueFrom:
+              secretKeyRef:
+                name: ${SECRET_NAME}
+                key: database-username
+          - name: DB_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: ${SECRET_NAME}
+                key: database-password
+          - name: NODE_REGION
+            value: ${NODE_REGION}
+          - name: NODE_TYPE
+            value: TE
+          - name: INSIGHTS_URL
+            value: "https://insights-qa.nuodb.com/api/1"
+          image: ${CONTAINER}
+          imagePullPolicy: IfNotPresent
+          name: te-${NODE_REGION}
+          ports:
+          - containerPort: 48004
+            protocol: TCP
+          resources: {}
+          terminationMessagePath: /dev/termination-log
+          terminationMessagePolicy: File
+          volumeMounts:
+          - mountPath: /archive
+            name: admin-volume-1
```

Comment:
Volume mounted at "/archive" for a TE. Also, the name is not parameterized by a variable like the volumes for nuoadmin and sm groups, which were parameterized by ${NODE_REGION}. Does this mean that all TEs get the same volume?